### PR TITLE
Update for v2022.1.1-alpha-1

### PR DESCRIPTION
**Pull request by prateekma**
- [ ] All artifacts are not downloading
- [ ] Headers bug (see Slack)
- [ ] C++ tools not showing up

### CODE REVIEW

**Comment by sciencewhiz** [comment]
Should update all the vscode extensions

File: scripts/vscode.gradle [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 def vscodeFile = file("$buildDir/vscodeConfig.json")
 
-def vsCodeVersion = '1.52.1'
+def vsCodeVersion = '1.60.1'
```
Comment (sciencewhiz) [inline scripts/vscode.gradle:4]:
```suggestion
def vsCodeVersion = '1.60.2'

```

**Comment by PeterJohnson** [comment]
distwrapper/gradle-wrapper.properties needs to be updated to 7.1.1

**Comment by PeterJohnson** [comment]
Superseded by #180.

**Pull request closed without merging**